.github/workflows/e2e-playright.yaml

```
@@ -1,4 +1,6 @@
-name: e2e
+# End-to-end testing that deploys and tests Supabase, API, UI, and VLLM
+
+name: e2e-playright
```

```suggestion
name: e2e-playwright
```
Minor spelling in a few places and in the name of the workflow